service.yml

```diff
@@ -10,3 +10,4 @@ github:
     repo_name: confluentinc/parallel-consumer
 semaphore:
     enable: true
+    run_semaphore_public: true
```

Comment:
i don't think this is the right place to allow this configuration.  it might make sense in `projects.json`, but since the team owning this repo can approve/merge a PR adding this config, it wouldn't really provide any roadblock.